core/src/CMakeLists.txt

```diff
@@ -70,14 +74,19 @@ if(Cabana_ENABLE_SILO)
   target_link_libraries(cabanacore INTERFACE SILO::silo)
 endif()
 
+if(Cabana_ENABLE_HDF5)
+  target_link_libraries(cabanacore INTERFACE ${HDF5_LIBRARIES})
```

Comment:
```suggestion
  target_link_libraries(cabanacore INTERFACE HDF5::HDF5)
```

Reply:
That's what I had originally, but I can't build with this change

Reply:
And with `hdf5::hdf5` (the c library) ?

Reply:
Looks like this was added/fixed somewhere between CMake 3.18 and 3.25. I can bisect if we want to bump the min for this change

Reply:
Added fixme note